src/services/transactions/sendBatchTransactions.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  BatchTransactionsResponseType,
+  SendBatchTransactionReturnType,
+  SignedTransactionType
+} from 'types';
+import axios from 'axios';
+import { TRANSACTIONS_BATCH } from 'apiCalls';
+
+export interface SendBatchTransactionsPropsType {
+  transactions: SignedTransactionType[] | SignedTransactionType[][];
+  account: string;
+  sessionId: string;
+  apiAddress: string;
+}
+
+export async function sendBatchTransactions({
+  transactions,
+  sessionId,
+  account,
```

Comment:
`account` refferes to the account object, it's better here to work with `address` (it's used like this trough the project)